### PR TITLE
fix unpause button

### PR DESCRIPTION
**Pull request by medyagh**
Screenshot 2023-01-23 at 2 23 26 PM

### CODE REVIEW

File: src/basicview.cpp [hunk 1]
```diff
@@ -90,9 +90,9 @@ static QString getPauseLabel(bool isPaused)
     return Constants::pauseIcon;
 }
 
-static QString getStartLabel(bool isRunning)
+static QString getStartLabel(bool isRunning,bool isPaused)
```
Comment (spowelljr) [inline src/basicview.cpp:93]:
```suggestion
static QString getStartLabel(bool isRunning, bool isPaused)
```

File: src/basicview.cpp [hunk 2]
```diff
@@ -135,8 +135,8 @@ void BasicView::update(Cluster cluster)
 #endif
     pauseButton->setText(getPauseLabel(isPaused));
     pauseButton->setToolTip(getPauseToolTip(isPaused));
-    startButton->setText(getStartLabel(isRunning));
-    startButton->setToolTip(getStartToolTip(isRunning));
+    startButton->setText(getStartLabel(isRunning,isPaused));
```
Comment (spowelljr) [inline src/basicview.cpp:138]:
```suggestion
    startButton->setText(getStartLabel(isRunning, isPaused));
```

**Review by spowelljr — CHANGES_REQUESTED** [review]
Could you also update this for advanced view?

**Comment by medyagh** [comment]
> Could you also update this for advanced view?

done

**Review by spowelljr — APPROVED** [review]

**Pull request merged**